src/Servers/HttpSys/src/NativeInterop/HttpApi.cs

```diff
@@ -136,18 +137,17 @@ private static void InitHttpApi(ushort majorVersion, ushort minorVersion)
             version.HttpApiMajorVersion = majorVersion;
             version.HttpApiMinorVersion = minorVersion;
 
-            var statusCode = HttpInitialize(version, (uint)HTTP_FLAGS.HTTP_INITIALIZE_SERVER, null);
+            var statusCode = HttpInitialize(version, (uint)(HTTP_FLAGS.HTTP_INITIALIZE_SERVER | HTTP_FLAGS.HTTP_INITIALIZE_CONFIG), null);
```

Comment:
What happen on older OSes that don't have this API?

Reply:
HTTP_INITIALIZE_CONFIG isn't a new flag, I think it's been around forever. What changed is that HttpIsFeatureSupported now requires it.